pkgs/development/tools/jdt-language-server/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, jdk15
+}:
+
+stdenv.mkDerivation rec {
+  pname = "jdt-language-server";
+  version = "0.67.0";
+  timestamp = "202012170459";
+
+  src = fetchurl {
+    url = "https://download.eclipse.org/jdtls/milestones/${version}/jdt-language-server-${version}-${timestamp}.tar.gz";
+    sha256 = "11qzwf2l737hdv1g7bfcrmdxcrl7g9z2k9nhy9lknlabmhq4mcdl";
+  };
+
+  sourceRoot = ".";
+
+  buildInputs = [
+    jdk15
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  installPhase =
+    let
+      # The application ships with config directories for linux and mac
+      configDir = if stdenv.isDarwin then "config_mac" else "config_linux";
+      # The application will store it's data here. Note especially the escaping so
+      # that the env vars are interpreted at runtime and not during installPhase.
+      runtimePath = "\\\${XDG_CACHE_HOME:-\$HOME/.cache}/jdt-language-server";
```

Comment:
I think it should be `\\\$HOME`, currently that line expands to `${XDG_CACHE_HOME:-/homeless-shelter/.cache}/jdt-language-server`

Reply:
Thank you! I updated this :-) :+1: